src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -41,35 +37,43 @@ public void RequestStart(bool isHttps, string scheme, string method, HostString
         // Tags must match request end.
         var tags = new TagList();
         InitializeRequestTags(ref tags, isHttps, scheme, method, host);
-        _currentRequestsCounter.Add(1, tags);
+        _activeRequestsCounter.Add(1, tags);
     }
 
-    public void RequestEnd(string protocol, bool isHttps, string scheme, string method, HostString host, string? route, int statusCode, Exception? exception, List<KeyValuePair<string, object?>>? customTags, long startTimestamp, long currentTimestamp)
+    public void RequestEnd(string protocol, bool isHttps, string scheme, string method, HostString host, string? route, int statusCode, bool unhandledRequest, Exception? exception, List<KeyValuePair<string, object?>>? customTags, long startTimestamp, long currentTimestamp)
     {
         var tags = new TagList();
         InitializeRequestTags(ref tags, isHttps, scheme, method, host);
 
         // Tags must match request start.
-        if (_currentRequestsCounter.Enabled)
+        if (_activeRequestsCounter.Enabled)
         {
-            _currentRequestsCounter.Add(-1, tags);
+            _activeRequestsCounter.Add(-1, tags);
         }
 
         if (_requestDuration.Enabled)
         {
-            tags.Add("protocol", protocol);
+            tags.Add("network.protocol.name", "http");
+            if (TryGetHttpVersion(protocol, out var httpVersion))
+            {
+                tags.Add("network.protocol.version", httpVersion);
+            }
+            if (unhandledRequest)
+            {
+                tags.Add("aspnetcore.request.is_unhandled", true);
+            }
 
             // Add information gathered during request.
-            tags.Add("status-code", GetBoxedStatusCode(statusCode));
+            tags.Add("http.response.status_code", GetBoxedStatusCode(statusCode));
             if (route != null)
             {
-                tags.Add("route", route);
+                tags.Add("http.route", route);
             }
             // This exception is only present if there is an unhandled exception.
-            // An exception caught by ExceptionHandlerMiddleware and DeveloperExceptionMiddleware isn't thrown to here. Instead, those middleware add exception-name to custom tags.
+            // An exception caught by ExceptionHandlerMiddleware and DeveloperExceptionMiddleware isn't thrown to here. Instead, those middleware add exception.type to custom tags.
             if (exception != null)
             {
-                tags.Add("exception-name", exception.GetType().FullName);
+                tags.Add("exception.type", exception.GetType().FullName);
```

Comment:
I think we have an open question here on the `exception.type` vs `http.error.reason` - https://github.com/lmolkova/semantic-conventions/pull/1#discussion_r1280908929

Reply:
There will definitely be `exception.type` to report app errors.

If `http.error.reason` is standardized then we can consider adding it in the future.